Display uncovered intermediates by default in item matrix (#285)
melexis/sphinx-traceability-extension: doc/usage.rst

    When given, an extra column that lists the intermediate item(s) per source item will be added between the columns
    that list sources and the linked targets. The argument will be used as title for this new column.

:onlycoveredintermediates: *optional*, *flag*

>     By default, all intermediate items are included. By providing the *onlycoveredintermediates* flag, only covered

Reviewer: ```suggestion
    By default, all intermediate items are shown. By providing this flag, only covered intermediate
```
